<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="hh" failures="1" tests="2" name="TestScreenshot" time="62.436" errors="0" timestamp="2022-03-03T15:55:57 IST">
  <testcase classname="com.qa.testCases.LoginPageTestScenario2" name="loginWithCorrectIDPass" time="16.909"/>
  <testcase classname="com.qa.testCases.LoginPageTestScenario2" name="loginWithIncorrectID" time="11.478">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.qa.testCases.LoginPageTestScenario2.loginWithIncorrectID(LoginPageTestScenario2.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginWithIncorrectID -->
</testsuite> <!-- TestScreenshot -->
